tests/components/sensor/test_command_line.py

```diff
@@ -68,3 +73,56 @@ def test_bad_command(self):
         data.update()
 
         self.assertEqual(None, data.value)
+
+    def test_update_with_json_attrs(self):
+        """Test attributes get extracted from a JSON result."""
+        data = command_line.CommandSensorData(
+            self.hass,
+            'echo { \\"key\\": \\"some_json_value\\" }', 15
```

Comment:
I would like to test more item in the JSON duct. Let's say you have 3 item in the output, and only pick up 2 item as json_attrs

Reply:
Done. Added two additional keys (and wanted both) to the original test, and added two more tests: One where the sensor config requests a key missing in the command's output, and one where the command's output has a key we don't request. I check in both cases that the key isn't in attributes.